tests/utils/osidb_cache.py

```diff
@@ -0,0 +1,44 @@
+import asyncio
```

Comment:
I would prefer to maintain the `osidb_cache` code at a single place.  We might consider also sharing the data, which comes with both advantages and disadvantages.  I can see that the content of `CVE-2025-0725.json` is different in each directory, I guess, because each of us captured the data using a different OSIDB instance.  In my case it was with `AEGIS_OSIDB_SERVER_URL="https://osidb-stage.prodsec.redhat.com/"`.